### ISSUE TITLE
The contents TOC has unusual space between the lines

### ISSUE THREAD

**Issue by kwaun** [readthedocs/sphinx_rtd_theme#931]
### Problem

This is an aesthetic bug. When I use a "contents" TOC I get some extra space in between. BTW the "toctree" TOC looks perfect. 
Looks like there are some unneeded "p" html tags.

### Reproducible Project

```rst
Sphinx sample
*************

.. contents::
    :local:

Section 1
=========

The TOC looks weird.

Subsection 1
------------

:(

Subsection 2
------------

Yep.

Subsubsection 1
^^^^^^^^^^^^^^^

Oh no.

Section 2
=========

It's true!

Subsection 3
------------

omg
```

#### Error Logs/Results

![theme-bug](https://user-images.githubusercontent.com/12612253/82597875-fd51db80-9ba9-11ea-85ed-48c00c7d9c80.png)

#### Expected Results

TOC with the same space as between subsection 1 and 2.

### Environment Info

- Python Version: 3.7.3
- Sphinx Version: 3.0.3
- RTD Theme Version: 0.4.3

**Comment by kwaun** [comment]
Or course same wrong behaviour when I use nested lists (bulleted or numbered).

**Comment by agjohnson** [comment]
Have you tried 0.5.0rc1 release of the theme? A number of the issues from the html5 writer are resolved in the 0.5.0rc1 release.

**Comment by kwaun** [comment]
Just tried. The issue is resolved in the 0.5.0rc1 release . Thanks!
